pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+
+@Library('pipelines-shared-library@RANCHER-1000') _
```

Comment:
Should it be RANCHER-1000?

Reply:
nope, its DRAFT mode, final one will be changed with 741
Draft mode to see design ok or no like this make